mdx-layout-css/src/index.tsx

```diff
@@ -15,7 +14,7 @@ export const CssLayout = ({ components = {}, onSwitch, ...rest }) => (
       }}
       onSwitch={onSwitch}
     />
-    <link rel="stylesheet" href={'../dist/commons.css'} />
+    <link rel="stylesheet" href={new URL('commons.css', import.meta.url).href} />
```

Comment:
I think it won't work outside of this when used via the packd bundler

Reply:
We still encountered this issue.
I think you will need to import it globally via `import '../src/commons.css'`

Reply:
> I think it won't work outside of this when used via the packd bundler

I know it won't work outside, but the purpose is to have work in the dockit examples, to override the effects of the tailwind css. In starters that use the dockit we don't need this.

Reply:
thx for the highlight 👍